### ISSUE TITLE
Accessability labels missing from remove from wallet screen

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1542]
### Description of problem

On iOS when a `?` is in the accessibility label it usually means it is missing but iOS is putting it its best guess. We need to go back and add them.

### Screenshots and/or log output

![IMG_0089](https://github.com/bcgov/bc-wallet-mobile/assets/390891/d9164492-bfb2-4c24-aa12-b79f07d0b104)

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

1252

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [ ] Low
